test/integration/integration_test.go

```diff
@@ -361,13 +377,11 @@ func selectFieldFromLastOutput(commandField string) string {
 		outputField = lastCommandOutput.StdErr
 	case "exitcode":
 		outputField = strconv.Itoa(lastCommandOutput.ExitCode)
-	default:
-		fmt.Errorf("Incorrect field type specified for comparison: %s", commandField)
 	}
 	return outputField
 }
 
-func shouldBeInValidFormat(commandField, format string) error {
+func shouldBeInValidFormat(commandField string, format string) error {
```

Comment:
@budhrg There was a suggestions above in the PR about using this format and not the shorted form. So I followed it and made it the same in whole file. Personally I like `name1, name2, name3 type` a little bit more :)